src/config/sharedConfig.ts

```diff
@@ -1,6 +1,8 @@
 // Generate your own WalletConnect 2 ProjectId here: https://cloud.walletconnect.com/app
 export const walletConnectV2ProjectId = '9b1a9564f91cb659ffe21b73d5c4e2d8';
 export const apiTimeout = 6000;
+export const faucetSettingEndpoint = '/faucet/settings';
```

Comment:
This could perhaps be declared after `faucetEndpoint` and instead have the value as `"${faucetEndpoint}/settings"`?